### PR TITLE
Adding comments

### PR DESCRIPTION
**Pull request by AtikaWaheed**

### CODE REVIEW

**Review by raeeschachar — CHANGES_REQUESTED** [review]
add 2 folders:
- tests
- pages
move test files to tests and page files to pages
you will have to fix imports accordingly too

File: Mercury_tour_page_file.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def flights_vertical(self):
+        # Move to flights vertical
+        self.driver.find_element_by_link_text('Flights').click()
+
+    def flight_details(self):
+        # submit form
+        self.driver.find_element_by_css_selector('input[value="oneway"]').click()
+        time.sleep(5)
+        passenger = self.driver.find_element_by_name('passCount')
+        departing_port = self.driver.find_element_by_name('fromPort')
```
Comment (raeeschachar) [inline Mercury_tour_page_file.py]: Please add separate methods for these

File: kayak_test_file.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import unittest
+import kayak_page_file
+from selenium.webdriver.common.keys import Keys
+from selenium import webdriver
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+import time
+
+
+class KayakMainSearch(unittest.TestCase):
```
Comment (raeeschachar) [inline kayak_test_file.py]: remove all Kayak related information from this

**Review by raeeschachar — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+*.pyc
+.xml
+.idea
```
Comment (raeeschachar) [inline .gitignore]: Add an empty line on end of every file, coding standards

File: Page.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+from selenium.webdriver.common.keys import Keys
+class BasePage(object):
```
Comment (raeeschachar) [inline Page.py]: Two empty lines before a class

File: Page.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    def matches_the_title(self):
+        return "Python" in  self.driver.title
+
+    def search_element_by_id(self):
+        element = self.driver.find_element_by_id('id-search-field')
+        element.send_keys("python")
+
+
+    def click_Go_button(self):
```
Comment (raeeschachar) [inline Page.py]: click_go_button, function name should be all small characters

File: Page.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        new_element = self.driver.find_element_by_id('submit')
+        new_element.click()
+
+    def is_result_found(self):
+        return "No results found." not in self.driver.page_source
+
+    def about_Page(self):
```
Comment (raeeschachar) [inline Page.py]: about_page

File: Page.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        self.driver.find_element_by_link_text('About').click()
```
Comment (raeeschachar) [inline Page.py]: empty line at end of file

File: pages/mercury_tour_page.py [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+class MainWelcomePage(BasePage):
+
+    def title_matches(self):
+        # Find out Header
+        return "Welcome: Mercury Tours" in self.driver.title
+
+    def login_cred(self):
+        # Give login cred and click submit
+        user_name = self.driver.find_element_by_name('userName')
+        user_name.send_keys("admin")
```
Comment (raeeschachar) [inline pages/mercury_tour_page.py]: username and password should be sent from the function you are calling this method, try something like:
```
def login_cred(self, username, password):
    user_name = self.driver.find_element_by_name('userName')       
    user_name.send_keys(username)
```

When you will call this function:
```
login_cred('admin', 'password_here')
```

File: pages/mercury_tour_page.py [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+        # Move to flights vertical
+        self.driver.find_element_by_link_text('Flights').click()
+
+    def pass_count(self):
+        passenger = self.driver.find_element_by_name('passCount')
+        Select(passenger).select_by_visible_text("4")
+
+    def from_port(self):
+        departing_port = self.driver.find_element_by_name('fromPort')
+        Select(departing_port).select_by_visible_text("Frankfurt")
```
Comment (raeeschachar) [inline pages/mercury_tour_page.py]: Same is the case for every function
``` 
def from_port(self, port):
    departing_port = self.driver.find_element_by_name('fromPort')
    Select(departing_port).select_by_visible_text(port)
```
When you will call this function: 
```
from_port('Frankfurt')
```

**Review by raeeschachar — CHANGES_REQUESTED** [review]
The comments I added are for all files and all functions. Please change them accordingly. Let me know if you don't understand anything

File: pages/mercury_tour_page.py (hunk 9 above)
Comment (AtikaWaheed) [reply pages/mercury_tour_page.py]: Done

File: pages/mercury_tour_page.py (hunk 8 above)
Comment (AtikaWaheed) [reply pages/mercury_tour_page.py]: Done

File: Page.py (hunk 4 above)
Comment (AtikaWaheed) [reply Page.py]: Done

File: .gitignore (hunk 3 above)
Comment (AtikaWaheed) [reply .gitignore]: Done

File: Page.py (hunk 5 above)
Comment (AtikaWaheed) [reply Page.py]: Done

File: Page.py (hunk 6 above)
Comment (AtikaWaheed) [reply Page.py]: Done

File: tests/geckodriver.log [hunk 10]
```diff
@@ -0,0 +1,49 @@
+1513231509297	geckodriver	INFO	geckodriver 0.19.0
```
Comment (raeeschachar) [inline tests/geckodriver.log]: add this file in gitignore too

File: tests/test file.py [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+        WebDriverWait(self.driver, 30)
+        assert main_page.is_result_found(), "No results found."
+        main_page.about_page()
+
+
+    def tearDown(self):
+        self.driver.close()
+
+if _name_ == "__main__":
+    unittest.main()
```
Comment (raeeschachar) [inline tests/test file.py:31]: empty line at the end of file

**Review by raeeschachar — CHANGES_REQUESTED** [review]

File: tests/test file.py (hunk 11 above)
Comment (AtikaWaheed) [reply tests/test file.py:31]: Done

**Review by raeeschachar — APPROVED** [review]

**Pull request merged**